helper_dart/lib/src/facet_list.dart

```diff
@@ -271,6 +285,7 @@ class _FacetList with DisposableMixin implements FacetList {
 
   @override
   void toggle(String value) {
+    _trackClickIfNeeded(value);
```

Comment:
This component can only present the data from the `facets` field of the search response (If I understand correctly, it is closest to the `RefinementList` component of the instansearch.js), so it's not possible to set numeric value range or filters and it might not fail.